tests/api/api/test_feature_sets.py

```diff
@@ -491,3 +532,207 @@ def test_features_list(db: Session, client: TestClient) -> None:
     assert (
         features_response["features"][1]["feature_set_digest"]["metadata"]["tag"] == tag
     )
+
+
+# There will be fields added (uid for example), but we don't allow any other changes
+def _assert_diff_contains_only_additions(original_object, new_object):
+    diff = DeepDiff(original_object, new_object, ignore_order=True,)
+    diff.pop("dictionary_item_added")
+    assert diff == {}
```

Comment:
In assertion functions prefer the terminology to be expected vs actual, so here `expected_object` and `actual_object` (or can be just `object_` (with trailing underscore cause it's built-in name))
This is too loose IMO, you can use `exclude_paths` exclude the field you expect to be enriched